pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -47,6 +47,8 @@ func (ds *dockerService) updateCreateConfig(
 	podSandboxID string, securityOptSep rune, apiVersion *semver.Version) error {
 	if networkMode := os.Getenv("CONTAINER_NETWORK"); networkMode != "" {
 		createConfig.HostConfig.NetworkMode = dockercontainer.NetworkMode(networkMode)
+	} else {
+		modifyHostNetworkOptionForContainer(false, podSandboxID, createConfig.HostConfig)
```

Comment:
This function is in `security_context.go`. Calling this in directly seems weird. I understand that this part of code for windows and linux are very different, maybe refactor/clean this up in the future.